packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -113,8 +113,8 @@ export class WalletService {
   walletId: string;
   copayerId: string;
   appName: string;
-  appVersion: string;
-  parsedClientVersion: { agent: string; major: number; minor: number };
+  appVersion: { agent?: string; major?: number; minor?: number };
+  parsedClientVersion: { agent?: string; major?: number; minor?: number };
```

Comment:
Should the IWalletService interface on line 79 be updated as well?

Reply:
Updated. It doesn't look like IWalletService is used anywhere...so I added an `implements` to the class.